src/detectors/utils/jsdetect.js

```diff
@@ -93,6 +93,6 @@ function scanScripts({ preferredScriptsArr, preferredCommand }) {
 module.exports = {
   hasRequiredDeps,
   hasRequiredFiles,
-  getYarnOrNPMCommand,
+  getYarnOrNPMCommand: getPackageManagerCommand,
```

Comment:
wondering why `getYarnOrNPMCommand` is still present if that function was deleted?

Reply:
As mentioned in the PR summary, `getYarnOrNPMCommand` was used on another file:
> I didn't changed the name of the exported function to avoid touching more files than needed. This should be changed at some point, probably. **Let me know if you'd want me to change it myself before merging.**